frontend/src/views/web/challenge-create.html

```diff
@@ -1,4 +1,20 @@
 <section class="ev-md-container text-center ">
+        <div class="loader-container card-loader-container" id="loader" ng-show="challengeCreate.isExistLoader" class="fade">
+            <div class="loader">
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
+        </div>
+    <div class="challenge-loader">
+      <div class="row">
+        <div class="col s12 m12">
+          <div class="show-loader">
+          </div>
+        </div>
+      </div>
```

Comment:
@HargovindArora There is no need to write loader again, please use loaderService for this. You only have to give the container for loader service and it will fade that container.
cc @RishabhJain2018

Reply:
@aka-jain I tried it in the controller without using this HTML code but it wasn't working!

Reply:
It's completely working fine now @RishabhJain2018 @aka-jain. Please do test it! :smile:

Reply:
@HargovindArora it would be a good idea to figure out why the loader wasn't working instead of duplicating the code for no reason. Sorry, we cannot merge the code till then.

Reply:
@deshraj Sure. I'll look into it! :+1: